packages/openapi-parser/src/utils/upgradeFromThreeToThreeOne.ts

```diff
@@ -6,7 +6,9 @@ import { traverse } from './traverse'
  *
  * https://www.openapis.org/blog/2021/02/16/migrating-from-openapi-3-0-to-3-1-0
  */
-export function upgradeFromThreeToThreeOne(specification: AnyObject) {
+export function upgradeFromThreeToThreeOne(originalSpecification: AnyObject) {
+  let specification = originalSpecification
```

Comment:
was the structured clone taking too much memory?

Reply:
It was causing errors in the pipeline syntax, looks like it’s relying on a variable reference 🫨, but I don’t want to tackle this in the same PR.